### DISCUSSION TITLE
X-Frame-Options is being set to SAMEORIGIN even if I set it to unset - Can't embed dashboard using iframe

### DISCUSSION THREAD

**Discussion by faizanvahevaria** [plausible/analytics#3367] [Self-Hosted Support]
I'm getting this error and can't embed dashboard using iframe.
Initially the error was:
Refused to display 'https://analytics.mywebsite.com/' in a frame because it set 'X-Frame-Options' to 'sameorigin'.

I'm trying to embed dashboard on a different website(domain) than the plausible self hosted domain.

Then I set 
Header always unset X-Frame-Options
in apache virtual host file for the analytics.mywebsite.com
Still getting same error

Then added 
Header always append X-Frame-Options "ALLOW-FROM https://123website.com/"

But still can't embed iframe on https://123website.com/analytics

Then I added
Header always set Content-Security-Policy "frame-ancestors 'self' https://123website.com/"
But still same error

Then I added
Header always set Content-Security-Policy "frame-ancestors *"

Same problem persists.

Please help.
I'm using Apache2 server for reverse proxy for docker compose up container.

**Comment by pau1phi11ips** [comment]
Are you sure it's Apache for the reverse proxy? I'm using nginx as the reverse proxy and I thought that was most common.
This worked in nginx:
`add_header Content-Security-Policy "frame-ancestors 'self' https://admin.mywebsite.com";`

I think Apache would be this, note the `;` is in a different place at the end
`Header set Content-Security-Policy "frame-ancestors 'self' https://admin.mywebsite.com;"`
